experimental/packages/otlp-proto-exporter-base/src/index.ts

```diff
@@ -13,7 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-export { OTLPProtoExporterNodeBase } from './OTLPProtoExporterNodeBase';
-export * from './types';
-export * from './util';
+export * from './platform';
```

Comment:
As above exports should be explicit and not implicit as this would export `potentially` internal values publically as part of the public API surface.